src/utils/ImageHelper.h

```diff
@@ -0,0 +1,36 @@
+#ifndef IMAGE_HELPER_H
+#define IMAGE_HELPER_H
+
+#include <vector>
+
+struct Pixel
+{
+	unsigned char R;
+	unsigned char G;
+	unsigned char B;
+	unsigned char A;
+};
+
+class Bitmap
```

Comment:
Let's rename the files to Bitmap.h and Bitmap.cpp and place them into the `src/client/graphics/` directory.

Reply:
Placed and renamed to Bitmap.